packages/anoncreds/src/utils/w3cAnonCredsUtils.ts

```diff
@@ -20,6 +20,7 @@ import {
   isUnqualifiedRevocationRegistryId,
   isIndyDid,
   getUnQualifiedDidIndyDid,
+  isUnqualifiedIndyDid,
```

Comment:
This is unrelated to this PR right? And probably fixed by some of the other PRs opened by @genaris?

Reply:
This got added further down in the file just for the case that the did is an unqualified indy did already. The code retrieves the unqualified indy did using the indy did but if it starts as an unqualified indy did the check fails. even though the `getUnqualifiedDidIndyDid` accounts for this case internally.